src/test/addrman_tests.cpp

```diff
@@ -5,6 +5,8 @@
 #include <test/setup_common.h>
 #include <string>
 #include <boost/test/unit_test.hpp>
+#include <util/asmap.h>
+#include <test/data/asmap.json.h>
```

Comment:
This doesn't look like json. Also, this file is 21 MB, a bit too much for the git repo.

You might be better off adding a makefile rule `%.bin.h: %.bin` (similar to `%.json.h: %.json`).

Reply:
For a big file that needs processing, maybe add a `contrib` script that fetches it (checking  the sha256 hash) and does its (determinstic) thing. And then just add the result to the repo.

Reply:
Right, I'll do both things.